graphql-server/go-server/graph/schema/entrypoint.graphqls

```diff
@@ -0,0 +1,34 @@
+type Query {
+    hello(name: String!): String!
```

Comment:
should be removed?

Reply:
No, this should be kept, if you just define an empty `query`, `mutation` will report an error.